content/administration/maintain/google_oauth.rst

```diff
@@ -0,0 +1,206 @@
+
+.. image:: google_oauth/create-credentials.png
+   :class: img-fluid
+   :align: center
+   :alt: Create Credentials on Google OAuth.
+
+- Under :guilabel:`Application Type`, select :guilabel:`Web Application` from the dropdown menu.
+- In the :guilabel:`Client ID` field, enter *Odoo*.
+- Under the :guilabel:`Authorized redirect URIs` label, click the button :guilabel:`ADD URI`, and
+  then input *https://yourdbname.odoo.com/google_gmail/confirm* in the :guilabel:`URIs 1` field. Be
+  sure to replace the *yourdbname* part of the URL with your actual Odoo database name.
+
+- Next, click on :guilabel:`Create` to generate an OAuth :guilabel:`Client ID` and
+  :guilabel:`Client Secret`. Finally, copy each generated value to a Notepad and navigate to the
+  Odoo database.
+
+.. image:: google_oauth/client-credentials.png
+   :class: img-fluid
+   :align: center
+   :alt: Client ID and Client Secret for Google OAuth.
+
+Setup in Odoo
+=============
+
+Enter Google Credentials
+------------------------
+
+First, open Odoo and navigate to the :guilabel:`Apps` module. Then, remove the
+:guilabel:`Apps filter` and type in *Google*. Install the module called :guilabel:`Google Gmail`.
+
+Next, navigate to the :menuselection:`Settings module --> General Settings`, and under the
+:guilabel:`Discuss` section, ensure that the checkbox for :guilabel:`Custom Email Servers` is
+checked. This populates a new option for :guilabel:`Gmail Credentials`. Make sure to check the box
+next to it. Then, copy and paste the :guilabel:`Client ID` and :guilabel:`Client Secret` into the
+respective fields and :guilabel:`Save` the settings.
+
+Configure outgoing email server
+-------------------------------
+
+After that, on the :guilabel:`General Settings` page, under :guilabel:`Custom Email Servers`, click
+:guilabel:`Outgoing Email Servers` to configure the external Gmail account.
+
+.. image:: google_oauth/outgoing-servers.png
+   :class: img-fluid
+   :align: center
+   :alt: Configure Outgoing Email Servers in Odoo.
+
+Then, :guilabel:`Create` a new email server and select the option for :guilabel:`Gmail`. Next, fill
+in the :guilabel:`Description` (can be anything) and the email :guilabel:`Username` and click on
+:guilabel:`Connect your Gmail account`.
+
+A new window from :guilabel:`Google` appears to complete the authorization process. Select the
+appropriate email address that is being configured in Odoo.
+
+If the email address is a personal account, an extra step pops up, click :guilabel:`Continue` to
+allow the verification and connect the Gmail account to Odoo.
+
+Then, allow Odoo to access the Google account by clicking on :guilabel:`Continue` or
+:guilabel:`Allow`. After that, the page navigates back to the newly configured outgoing email
+server in Odoo. The configuration automatically loads the token in Odoo, and a tag stating
+:guilabel:`Gmail Token Valid` appears in green.
+
+.. image:: google_oauth/green-token.png
```

Comment:
This image is too wide at 1430 px. Can you retake this screenshot @tiku-odoo ? You will probably have to shorten the width of the browser window a bit more to keep it under the tablet breakpoint width (768 px). And remember to compress your PNG image file after you take the screenshot!